nonce.go

```diff
@@ -72,10 +72,13 @@ func nonceForChunkSecretBox(i encryptionBlockNumber) Nonce {
 // above. There's almost certainly no harm in using the same nonces here as
 // above, since the encryption keys are ephemeral and the signatures already
 // have their own context, but at the same time it's a good practice.
-func nonceForChunkSigncryption(i encryptionBlockNumber) Nonce {
+func nonceForChunkSigncryption(i encryptionBlockNumber, isFinal bool) Nonce {
```

Comment:
We had talked about including AD in the encryption layer, so that the header hash could go in along with the final flag, to make sure it gets authenticated in anonymous sender mode. Did you change your mind about that one?